src/scripts/main.js

```diff
@@ -1,8 +1,20 @@
 'use strict';
 
 const people = require('./lib/people');
+const table = document.querySelector('.dashboard');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+people.forEach(p => {
```

Comment:
don't forget about destructurization 

```suggestion
people.forEach(({ name, sex, born, ... }) => {
```